src/pyproject_hooks/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""Wrappers to build Python packages using PEP 517 hooks
+"""
+
+from ._impl import (
+    BackendInvalid,
+    BackendUnavailable,
+    BuildBackendHookCaller,
+    HookMissing,
+    UnsupportedOperation,
+    default_subprocess_runner,
+    quiet_subprocess_runner,
+)
+
+__version__ = '0.13.0'
```

Comment:
Wouldn't this call for either a major version bump or versioning reset, since this practically makes this a new project from an end-user perspective?

Reply:
I don't think it's required, because with the name change there's no risk of unwittingly upgrading. But I'd be happy enough to call it 1.0 now that we're cutting off the deprecated bits of API.